packages/react-html/src/components/tests/Style.test.tsx

```diff
@@ -4,17 +4,32 @@ import {mount} from '@shopify/react-testing';
 import {Style} from '../Style';
 
 describe('<Style />', () => {
-  it('renders attributes', () => {
-    const style = mount(
-      <Style href="foo.css" integrity="00000000" crossOrigin="anonymous" />,
-    );
+  describe('<link />', () => {
+    it('renders attributes', () => {
+      const style = mount(
+        <Style href="foo.css" integrity="00000000" crossOrigin="anonymous" />,
+      );
 
-    expect(style).toContainReactComponent('link', {
-      href: 'foo.css',
-      type: 'text/css',
-      rel: 'stylesheet',
-      integrity: '00000000',
-      crossOrigin: 'anonymous',
+      expect(style).toContainReactComponent('link', {
+        href: 'foo.css',
+        type: 'text/css',
+        rel: 'stylesheet',
+        integrity: '00000000',
+        crossOrigin: 'anonymous',
+      });
+    });
+  });
+
+  describe('<style />', () => {
+    it('renders attributes', () => {
+      const children = 'hello world';
+      const style = mount(<Style id="foo">{children}</Style>);
```

Comment:
I would prefer for the test to reflect how something should be use.
maybe use the inline style you had mentioned in the description?